dotnet/src/IntegrationTests/IntegrationTests.csproj

```diff
@@ -11,6 +11,10 @@
     <UserSecretsId>b7762d10-e29b-4bb1-8b74-b6d69a667dd4</UserSecretsId>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="Plugins\instacart-ai-plugin.json" />
```

Comment:
Why is that here?

Reply:
It's used during the test that we can successfully load the plugin from a stream, I think a byproduct of how VS handles Content with copy-always:

<img width="239" alt="image" src="https://github.com/microsoft/semantic-kernel/assets/146438/2c541931-7d02-4698-9be2-ae14367f162f">